### ISSUE TITLE
A question about how to conduct customized training

### ISSUE THREAD

**Issue by Brembles** [rohitgandikota/erasing#21]
Your work is fantastic. If I want to train a model to specifically remove a concept similar to Van Gogh, how many epochs would be required ？

**Comment by rohitgandikota** [comment]
Thanks for your interest! I believe somewhere around 500-1000 epochs should do the job. If you are using the source code from our demo (diffusers version), then I would suggest 200 epochs. If you are interested in our demo, it is linked on our project website.

**Comment by Brembles** [comment]
Thank you very much for your guidance! When I was training using your model, I only used Van Gogh as the prompt, and also set words as the prompt. The learning rate was set to 1e-5 and I only trained the xattn. I trained for 200 epochs at a time, and once for 500 epochs, but the results were not as good as the checkpoint you provided. How did you do it?

**Comment by rohitgandikota** [comment]
We trained it exactly the way you described. When you say "results are not as good" - how do you mean? Could you share some results?

**Comment by Brembles** [comment]
![38_0](https://github.com/rohitgandikota/erasing/assets/132242633/d2e50c2d-23a5-4a1b-9c75-4954e9d39258)
![38_0](https://github.com/rohitgandikota/erasing/assets/132242633/71126352-0732-48e4-be74-8e605dc95eb5)
What I mean is that my checkpoint doesn't completely remove the concept of "Van Gogh" under certain prompts. Could you share your training parameters with me?

**Comment by rohitgandikota** [comment]
Aah, so the ESD model is not loaded properly! clearly they look very close to each other. We used the default parameters from the training script we provided. I would look at the file name and it is being loaded properly. In the evaluation script before you do `.load()` please check if the file it is trying to load exists. by doing `os.path.exists(model_path)` .

**Comment by Brembles** [comment]
The checkpoint I published is the one I trained. Your checkpoint performs very well, so I think my model loading is correct. I don't understand where I went wrong with my training parameters. Currently, I have set train-epoch=500 and lr=1×10^-5. Could you tell me your training parameters?

**Comment by rohitgandikota** [comment]
As I mentioned, we use the default parameters in the training script - LR 1e-5 and 1000 epochs

**Comment by Brembles** [comment]
Thank you for your guidance, I will try again!

**Comment by Brembles** [comment]
I retrained the model but the results are still not good. Maybe you could give me your email address, so I can contact you through email.

**Comment by leonardoooooooooooooooooooooooooooo** [comment]
having the same issue, the output data is not good as in your paper, not removing concept but just erase quality images, any suggestions? I m doing this on flux with this function https://github.com/tomguluson92/Erasing_Flux/blob/main/train_flux_esd.py

**Comment by rohitgandikota** [comment]
Hi all, 

We released a latest code update. This should make it easier for training and much efficient for SDv1.4 and SDXL. Hope this helps. Please feel free to reopen this thread if the issue still persists
